backend/config/config.go

```diff
@@ -228,6 +228,31 @@ func NewConfig(appConfigFilename string, accountsConfigFilename string) (*Config
 	return config, nil
 }
 
+// SetBtcOnly sets non-bitcoin accounts in the config to false
+func (config *Config) SetBtcOnly() {
+	config.appConfig.Backend.LitecoinP2WPKHP2SHActive = false
+	config.appConfig.Backend.LitecoinP2WPKHActive = false
+	config.appConfig.Backend.EthereumActive = false
+}
+
+// BaseBtcConfig sets the TBTC configuration to the provided electrumIP and electrumCert
+func (config *Config) BaseBtcConfig(electrumIP, electrumCert string, network string) {
```

Comment:
To keep the separation of concerns, it's better to have methods here like `SetBTCElectrumServers`, `SetTBTCElectrumServers`, and do the logic of which to change how in the bitboxbase folder.